example/android/app/src/main/AndroidManifest.xml

```diff
@@ -11,7 +11,7 @@
             android:configChanges="orientation|keyboardHidden|keyboard|screenSize|smallestScreenSize|locale|layoutDirection|fontScale|screenLayout|density|uiMode"
             android:hardwareAccelerated="true"
             android:windowSoftInputMode="adjustResize"
-            android:enabled="false"
+            android:enabled="true"
```

Comment:
Why would you need to enable MainActivity in the manifest just to disable it later in kotlin?🤨

Reply:
because it is not actually disabled from here (will Generate a new App icon in app list )
so, I thought to disable it from the kotlin.

Reply:
don't Worry Bro I, have Found a other way just Verify it in issues comment.